bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/pump.xml

```diff
@@ -91,9 +91,9 @@
 
 	<channel-type id="pumpLastSpeed">
 		<item-type>Number:Dimensionless</item-type>
-		<label>Last Speed</label>
-		<description>Last Speed (%)</description>
-		<state pattern="%d %unit%" readOnly="true"/>
+		<label>Last Speed % </label>
+		<description>Last Speed %</description>
+		<state pattern="%1.0f %" readOnly="true"/>
```

Comment:
I see you use systematically %1.0f. I have a doubt what means the initial 1. Do you know ?

Reply:
That is how I've always seen it done.  Knowing I don't need any decimal places, I changed these all to %d